package.json

```diff
@@ -62,15 +62,15 @@
     "@vitejs/plugin-vue": "^5.0.3",
     "glob": "^8.0.3",
     "install": "^0.13.0",
-    "lodash": "*",
+    "jsdom": "^24.0.0",
+    "lodash": "^4.17.21",
```

Comment:
`lodash` should be peerDependency

Reply:
added lodash to peerDependency and removed from devDependencies